pytorch_lightning/loggers/base.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+import torch
```

Comment:
the imports are usually structures following way (each block is separated by a blank line)
1. native libraries like os, argparse
2. third-party libs like torch
3. this package imports pytorch_lightning

Reply:
@monney could you pls check the imports